config_builder.py

```diff
@@ -68,6 +68,7 @@ def build_datadog_conf(self):
         self.set_from_env_mapping('DD_URL', 'dd_url')
         self.set_from_env_mapping('STATSD_METRIC_NAMESPACE', 'statsd_metric_namespace')
         self.set_from_env_mapping('USE_DOGSTATSD', 'use_dogstatsd')
+        self.set_from_env_mapping('HISTOGRAM_PERCENTILES', 'histogram_percentiles')
```

Comment:
👍 on this one. Although, could you rename it to `DD_HISTOGRAM_PERCENTILES` to be forward compatible with agent6?
https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config.go#L99